### PR TITLE
Weekly PR

### PR DESCRIPTION
**Pull request by vedi**

### CODE REVIEW

File: src/documents/android/profile/Profile_GettingStarted.html.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ platform: 'android'
 
 ##Getting Started
 
-1. Download [android-profile](http://library.soom.la/fetch/android-profile/1.0.1?cf=github).
+1. Download [android-profile](http://library.soom.la/fetch/android-profile/latest?cf=github).
```
Comment (borisspektor) [inline src/documents/android/profile/Profile_GettingStarted.html.md]: @vedi shouldn't this also be cf=knowledge base?

File: src/documents/cocos2dx/cpp/grow/GrowSpend_GettingStarted.html.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Go to the [GROW dashboard](http://dashboard.soom.la) and sign up \ login. Upon l
   soomla::CCSoomla::initialize("ExampleCustomSecret");
   ```
 
-  <div class="warning-box">Choose this secret wisely, you can't change it after you launch your game!</div>
```
Comment (borisspektor) [inline src/documents/cocos2dx/cpp/grow/GrowSpend_GettingStarted.html.md]: @vedi why delete this line?

File: src/documents/cocos2dx/cpp/store/Store_GettingStarted.html.md [hunk 3]
```diff
@@ -153,31 +153,6 @@ In your XCode project, perform the following steps:
     </application>
     ```
 
-
-##**Working with sources**
```
Comment (borisspektor) [inline src/documents/cocos2dx/cpp/store/Store_GettingStarted.html.md]: @vedi why did you remove this section?

File: src/documents/android/profile/Profile_GettingStarted.html.md (hunk 1 above)
Comment (vedi) [inline src/documents/android/profile/Profile_GettingStarted.html.md]: @borisspektor, sure, it was a merging

File: src/documents/cocos2dx/cpp/grow/GrowSpend_GettingStarted.html.md (hunk 2 above)
Comment (vedi) [inline src/documents/cocos2dx/cpp/grow/GrowSpend_GettingStarted.html.md]: @borisspektor, my fault, bad conflict resolving
